test/ReadsUtils_server_test.py

```diff
@@ -680,6 +687,46 @@ def test_FASTQ_val_fail_bad_ext(self):
                             'File data/sample.txt is not a FASTQ file')
 
     # Upload tests ########################################################
+    def test_upload_fail_min_len_reads(self):
+
+        # The min_Sample.fastq file contains only one read with a length of 1.
+        read_length = 1
+
+        self.fail_upload_reads(
+            {'fwd_file': 'data/min_Sample.fastq',
+             'sequencing_tech': 'seqtech',
+             'wsname': self.ws_info[1],
+             'name': 'filereads1',
+             'min_read_length': read_length + 1},
+            "Invalid FASTQ file - Path: /kb/module/test/data/min_Sample.fastq."
+            )
+
+    def test_upload_min_len_reads(self):
```

Comment:
There should be at happy and sad tests that handle a case where the min length is something other than 1

and with > 1 read / file

Reply:
I did a `test_upload_fail_min_len_reads` (sad) and `test_upload_min_len_reads` (happy) test. Read length should be matter here. What cases are not covered in your option? I actually feel this model is quite over tested.

Reply:
A case other than when min len reads is == 2 and reads / file >  1

Reply:
I made the test file having 2 reads now.

Reply:
well... that's good but now I think you should add tests with reads with a length of 1 and do the same tests.

Reply:
👍